frontend/src/metabase-types/types/Query.ts

```diff
@@ -302,14 +302,23 @@ export type ExpressionClause = {
   [key: ExpressionName]: Expression;
 };
 
-export type Expression = [
-  ExpressionOperator,
-  ExpressionOperand,
-  ExpressionOperand,
-];
+export type Expression =
+  | NumericLiteral
```

Comment:
We should add `BooleanLiteral` and `StringLiteral` here as well. And also adjust `ExpressionOperand`. Attaching screenshots with examples of such expressions - take a look at the state in the react dev tools 

<img width="1570" alt="Screenshot 2023-03-01 at 16 47 03" src="https://user-images.githubusercontent.com/8542534/222174626-c3536592-f36d-4f75-afe1-8bfdbd44d3fd.png">
<img width="1728" alt="Screenshot 2023-03-01 at 16 47 21" src="https://user-images.githubusercontent.com/8542534/222174656-4640e223-ff8a-450c-9b9a-a5c614a07aa2.png">